src/Hostnet/Sniffs/Functions/ReturnTypeDeclarationSniff.php

```diff
@@ -181,23 +188,23 @@ private function getCharacterAfterReturnTypeDeclaration(PHP_CodeSniffer_File $ph
     /**
      * Fix the spacing between start and end
      *
-     * @param PHP_CodeSniffer_File $phpcs_file The file being scanned.
-     * @param array $tokens Token stack for this file
+     * @param File   $phpcs_file The file being scanned.
+     * @param array  $tokens Token stack for this file
      * @param string $required_spacing Required spacing between start and end
-     * @param int $start Position of the start in the token stack
-     * @param int $end Position of the end in the token stack
+     * @param int    $start Position of the start in the token stack
+     * @param int    $end Position of the end in the token stack
      *
      * @return void
      */
-    private function fixSpacing(PHP_CodeSniffer_File $phpcs_file, $tokens, $required_spacing, $start, $end)
+    private function fixSpacing(File $phpcs_file, $tokens, $required_spacing, $start, $end)
     {
         //insert whitespace if there is no whitespace, and whitespace is required
         if (($start + 1) === $end && empty($required_spacing) === false) {
             $phpcs_file->fixer->addContent(
                 $start,
                 $required_spacing
             );
-        //otherwise, if there is whitespace, change the whitespace to the required spacing
+            //otherwise, if there is whitespace, change the whitespace to the required spacing
```

Comment:
I think this comment should be one line lower, so it follows the `} else {` statement below.